utils.js

```diff
@@ -23,3 +23,20 @@ export const section = (...args) => {
 };
 
 export const isUserEnvironment = !process?.env?.CI && !process?.env?.CIRCLE_JOB;
+
+export const compareVersions = (v1, v2) => {
```

Comment:
Maybe using https://www.npmjs.com/package/semver might be safer here?

Reply:
Semver package is quite small, it might simplify this part of the code too. I wouldn't mind trying it out for safety in the checks

Reply:
I agree, just didn't want to add a new dependency in my very first commit 😅

Reply:
Ha! yes agreed. Small enough it's worth it I think.